pennylane/_device.py

```diff
@@ -51,7 +51,22 @@ class Device(abc.ABC):
     """
 
     # pylint: disable=too-many-public-methods
-    _capabilities = {}  #: dict[str->*]: plugin capabilities
+    _capabilities = {
+        'model': None,
+        'passthru_interface': None,
+        'supports_reversible_diff': False,
+        'supports_exact': False,
+        'supports_sampled': False,
+        'supports_inverse_operations': False,
+        'supports_tensor_observables': False,
```

Comment:
I decided to go with "supports" since it gives a clear hint at the boolean type of the value.